### ISSUE TITLE
TextBox issues

### ISSUE THREAD

**Issue by jinek** [jinek/Consolonia#184]
1) When typing text into TextBox the caret blinks in an unusual way (too fast in my case). it looks like it reset by the timer additionally to blink by the terminal itself. It seems related to upgrade to newer Avalonia

2) When focusing on Mid-aligned and right-aligned textboxes, the initial caret position is set wrong

3) MultiLine Textbox seems  does not wrap text anymore. Navigating to the right side from visible area seems not possible

**Comment by tomlm** [comment]
this seems to work on Windows. Is this linux/macos?

**Comment by jinek** [comment]
> this seems to work on Windows. Is this linux/macos?

Yes, on fedora. Now I'm also thinking how to easily test on windows.  Virtual machine would be enough,  but windows has strict hardware requirements.  Do i need windows 11 for testing?

**Comment by tomlm** [comment]
You can use windows 10, but you definitely need to turn on Windows Terminal. It's built into 11, but I think you need to enable it on 10.
https://learn.microsoft.com/en-us/windows/terminal/install 

Windows Terminal is where they have full support for ANSI escape codes

**Comment by tomlm** [comment]
I would be surprised if you couldn't run in VirtualBox.
